index.bs

```diff
@@ -1712,6 +1712,9 @@ This [=internal method=] accepts three arguments:
     ::  This argument is a Boolean value which is [TRUE] if and only if the caller's [=environment settings object=] is
         [=same-origin with its ancestors=]. It is [FALSE] if caller is cross-origin.
 
+        Note: Invocation of this [=internal method=] indicates that it was allowed by
+        [=permissions policy=], which is evaluated at the [[!CREDENTIAL-MANAGEMENT-1]] level.
+        See [[#sctn-permissions-policy]].
```

Comment:
This is intended so it appears related to `sameOriginWithAncestors`, is that intentional?

Reply:
I was matching `5.1.4.1. PublicKeyCredential’s [[DiscoverFromExternalSource]](origin, options, sameOriginWithAncestors) Method`, though I agree that the indentation is strange. Let me know if you'd like it changed in both places, otherwise I will leave it as-is for now.

Reply:
Ah, I see. Let's handle those both together later then, if at all.